fe2-android/docs/README.md

```diff
@@ -165,13 +165,61 @@ This is the current organization, but as this project evolves you should feel fr
 
 For more information on linking up together the user interface with the application state, please make sure you have a solid understanding of the [MVVM pattern](#application-design) and look at the `ViewModel` classes and the `LAORepository`.
 
+### Dependency Injection
 
-## Testing
+The project uses dependency injection to provide the different dependencies instances. If you don't know what it is, android provides a [simple guide](https://developer.android.com/training/dependency-injection)
```

Comment:
There is a missing point after the link. Same thing for line 181.

Reply:
Thanks, I will change that